azkaban-web-server/src/main/less/base.less

```diff
@@ -7,8 +7,8 @@
 }
 
 .container-fill {
-  position: absolute;
-  top: 230px;
+  position: relative;
```

Comment:
This tag looks being used by many other pages, like flowpage, jobdetailpage. Have you checked that everywhere is fine?

Reply:
Thanks for pointing it out; I have checked the other tabs an there was impact.

Moved all the styling logic to `div#graphView.container-fill`, which is specific to the graph pane, and tested the updated version on chrome and firefox, it looks good to me now.